rplugin/python3/deoplete/filter/converter_reorder_kind.py

```diff
@@ -0,0 +1,50 @@
+# ============================================================================
+# FILE: converter_reorder_kind.py
+# AUTHOR: @reaysawa
+# License: MIT license
+# ============================================================================
+
+from deoplete.filter.base import Base
+
+
+class Filter(Base):
+    def __init__(self, vim):
+        super().__init__(vim)
+
+        self.name = 'converter_reorder_kind'
+        self.description = 'Reorder candidates based on their kind'
+
+    def filter(self, context):
+        preferred_order = self.vim.call(
+            'deoplete#custom#_get_option', 'kind_order_preference'
```

Comment:
It should be `deoplete#custom#_get_filetype_option`.

Reply:
how do I specify options for filters?